docs/docs/index.md

```diff
@@ -1,8 +1,21 @@
-# `sbi`: simulation-based inference
+# `sbi`: simulation-based inference toolkit
 
-`sbi` is a toolbox that let you run simulation-based inference methods simply using a high level API:
+`sbi` provides access to simulation-based inference methods via a user-friendly
+interface:
 
-![using sbi](static/infer_demo.gif)
+```python
+# simulation
+theta = prior.sample((1000,))
+x = simulator(theta)
+
+# training
+inference = SNPE(prior).append_simulations(theta, x)
+inference.train()
+
+# inference
+posterior = inference.build_posterior()
+posterior_samples = posterior.sample((1000,), x=x_o)
+```
```

Comment:
I think it's time to remove the GIF. It just changes to fast and I believe it's not really helpful. I suggest to replace it with an actual code snippet.

Reply:
I like the idea of an actual code snipet! I think this helps onboarding a lot.
I suggest to add a dummy prior and simulator so this code is actually runnable, it helps a lot understanding the base concept of the toolkit.